.travis.yml

```diff
@@ -15,17 +15,11 @@ env:
 matrix:
   fast_finish: true
   include:
-    - php: 7.0
-      env:
-        - DEPS=lowest
-    - php: 7.0
+    - php: 7.1
```

Comment:
There are still 2 tests with the same environment (php 7.1) with `DEPS=locked`.
`EXECUTE_CS_CHECK=true` and `EXECUTE_TEST_SCRUTINIZER=true` should be added in the already existing enviroment.